ios/FluentUI/Core/Theme/Tokens/GlobalTokens.swift

```diff
@@ -71,6 +71,36 @@ public class GlobalTokens: NSObject {
         }
     }
 
+    // MARK: - GradientColor
+
+    @objc(MSFGradientColorToken)
+    public enum GradientColorToken: Int, TokenSetKey {
```

Comment:
I think I'd prefer these to be in the `BrandColorToken` section, so that anything for a brand alias token is coming from there, but I'm not too picky.

Reply:
The issue is that the `brandColor` function that uses `BrandColorToken` returns a UIColor and not a `ColorValue`. The token could be added to `BrandColorsToken` though to get a `ColorValue` but there is a deprecated mark over that section so I decided against adding anything in there.

Reply:
Ah, there's two different brand color sections, since we haven't finished the clean up for moving to the system types.